app/models/host.rb

```diff
@@ -92,6 +92,8 @@ class Host < ApplicationRecord
   has_many                  :cloud_services, :dependent => :nullify
   has_many                  :host_cloud_services, :class_name => "CloudService", :foreign_key => "host_id",
                             :inverse_of => :host
+  has_many                  :host_host_aggregates
```

Comment:
when this is not HABTM, the naming convention is not strict

so it could be something more readable like host_aggregate_hostsh